frontend/src/utilities/valueUnits.ts

```diff
@@ -155,3 +155,17 @@ export const isMemoryLimitLarger = (
 
   return isLarger(limitMemory, requestMemory, MEMORY_UNITS_FOR_PARSING, isEqualOkay);
 };
+
+export const formatMemory = (value?: ValueUnitString): ValueUnitString | undefined => {
+  if (!value) {
+    return;
+  }
```

Comment:
If the value is undefined or an empty string, return itself.
```suggestion
  if (!value) {
    return value
  }
```

Reply:
Thanks @christianvogt ! Updated